docs/querying/basics.md

```diff
@@ -106,6 +106,11 @@ Example:
     'these are unescaped: \n \\ \t'
     `these are not unescaped: \n ' " \t`
 
+This can be useful for [`regex expressions`](#regular-expressions) on the regex-match selectors, for example:
+
+    {le=~"0\\.5"}
```

Comment:
These examples illustrate using escaping in label matcher label value regexes, but this is the section about PromQL string literals. The "strings" in label matchers are not a string literal in the PromQL sense, but just a syntactic part of a vector selector node (not their own expression). So I would either move this to wherever label matchers are explained, or use a different example here where you have to escape something in a PromQL string literal - probably only in the case of the `label_replace()` function regex.